docs/index.rst

```diff
@@ -0,0 +1,134 @@
+
```

Comment:
Is this file auto-generated from the README? Or do we need to manually keep it up to date?

Reply:
Good question. That's a concern I have too. I would expect that in all likelihood, the README will eventually just point to the documentation on RTD.

Reply:
OK, that seems like a good solution to me.